utils/api.go

```diff
@@ -52,7 +52,7 @@ func RenderWebError(w http.ResponseWriter, r *http.Request, status int, params u
 		http.Error(w, "", http.StatusInternalServerError)
 		return
 	}
-	destination := strings.TrimRight(GetSiteURL(), "/") + "/error?" + queryString + "&s=" + base64.URLEncoding.EncodeToString(signature)
+	destination := "/error?" + queryString + "&s=" + base64.URLEncoding.EncodeToString(signature)
```

Comment:
This is a behaviour change, right? If the user accessed the site on the `siteURL`, it's not a problem, but is there ever a reason to force the user back to `siteURL` as occurred before?

Reply:
I changed it from relative to absolute pretty recently here: https://github.com/mattermost/mattermost-server/pull/8178/files

At the time I though relative URLs weren't up to spec, which was true as of [RFC 2616](https://tools.ietf.org/html/rfc2616#section-14.30), but isn't as of [RFC 7231](https://tools.ietf.org/html/rfc7231#section-7.1.2) (which obsoletes 2616). So I'm just changing it back since it's easier.